chips/nrf52840/src/ieee802154_radio.rs

```diff
@@ -1347,7 +1347,9 @@ impl<'a> kernel::hil::radio::RadioData<'a> for Radio<'a> {
             return Err((ErrorCode::OFF, buf));
         } else if self.busy() {
             return Err((ErrorCode::BUSY, buf));
-        } else if buf.len() < radio::PSDU_OFFSET + frame_len + radio::MFR_SIZE {
+        } else if !(self.state.get() == RadioState::ACK)
```

Comment:
Because ACK frames don't need a CRC, PHR, or reserved byte? If so, please make explicit in a comment here so the next person knows why this condition is OK.